src/test/java/org/ays/utility/DataProvider.java

```diff
@@ -172,4 +172,17 @@ public static Object[][] invalidPhoneNumberDataForFilter() {
         };
     }
 
+    @org.testng.annotations.DataProvider(name = "statusTransitions")
+    public static Object[][] statusTransitions() {
+        return new Object[][]{
+                {"READY"},
+                {"IDLE"},
+                {"BUSY"},
+                {"IDLE"},
+                {"READY"},
+                {"BUSY"},
+                {"READY"}
+        };
+    }
```

Comment:
Burası için `UserSupportStatus` diye bir enum oluşturmuştuk, onu kullanabiliriz

Reply:
Enum kullanıldı.